release_tester/selenium_ui_test/pages/views_page.py

```diff
@@ -354,6 +337,152 @@ def create_improved_views(self, view_name, types):
         max_buffer_size_sitem.send_keys("33554434")
         time.sleep(2)
         self.webdriver.refresh()
+    
+    def checking_modified_views(self, deployment):
+        """This method will check views for 3.10.x package version"""
+        print('Selecting improved views \n')
+        views = "//*[text()='improved_arangosearch_view_01']"
+        views_sitem = self.locator_finder_by_xpath(views)
+        views_sitem.click()
+        time.sleep(2)
+        self.wait_for_ajax()
+
+        print('Selecting Consolidation Policy \n')
+        policy = "//*[text()='Consolidation Policy']"
+        policy_sitem = self.locator_finder_by_xpath(policy)
+        policy_sitem.click()
+        time.sleep(2)
+
+        self.wait_for_ajax()
+        print('Selecting segments min value \n')
+        segment_min = "/html/body/div[2]/div/div[2]/div[2]/div/div/div[1]/div/div/table/tbody/tr[2]/th[2]/input"
```

Comment:
there is no other way than this lengthy xpath?
It would as well be prefered, if you find a surrounding element that is better identifieable, and go from there up/down the xpath fewer steps.